src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/LogSpec.groovy

```diff
@@ -85,9 +85,10 @@ class LogSpec extends GrpcBaseSpecification {
         then: "Current remote log server configuration is deleted"
         response.deleted
 
+        /* https://github.com/telstra/open-kilda/issues/5408
```

Comment:
A minor thing: I'd recommend to use `*` on all lines of the multiline comment, in the same manner as in copywrite.

Reply:
I'll do it in the next PR's. I wouldn't like to re-start PR regression for one cosmetic fix.